traffic_router/core/src/test/java/com/comcast/cdn/traffic_control/traffic_router/core/config/ConfigHandlerTest.java

```diff
@@ -282,4 +315,113 @@ public void testParseLocalizationMethods() throws Exception {
         actual = Whitebox.invokeMethod(handler, "parseLocalizationMethods", "foo", CZMethodsJson);
         assertThat(actual, equalTo(expected));
     }
+
+    @Test
+    public void parseRegionalGeoConfigNegative() throws Exception {
+    	final long interval = 60000000000l;
```

Comment:
something terrible happened to the formatting in this file! Oh no! :)

Reply:
done